src/test/resources/net/sf/jabref/logic/exporter/BibTeXMLExporterTestInvalidInbook.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<file xmlns="http://bibtexml.sf.net/">
+    <entry id="Mustermann2016">
+        <inbook>
+            <bibtexkey>Mustermann2016</bibtexkey>
```

Comment:
Only inbook has the field key, that can be set. You can look at the generated Inbook class, there's a field key. Also in the other test for Inbook there's a bibtexkey as well.